### DISCUSSION TITLE
long time didn't get the Github Copilot Access after approved the student verificaion

### DISCUSSION THREAD

**Discussion by xjl-0806** [community/community#113391] [GitHub Education]
### Hi everyone,

How long after I’ve been approved will I receive my academic benefits?
Congrats, your request has been approved and your benefits will be available soon!
First off, congratulations on being approved.  When you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away.
We aim to provide you with your educational benefits within 3 days of this approval status. However, during busy periods like the start of a new semester, it may take a bit longer.
If you have not received your education benefits within 5 days of your approval status please raise a discussion on the GitHub Education community and the Education community team will be able to assist in resolving this for you. 
You’ll get an email notification once your academic benefits are ready, or you can check your status here . Meanwhile, as a verified GitHub Global Campus student, you can explore the platform, connect with a local Campus Expert , attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from GitHub Community Exchange and much more!

**Comment by samus-aran** [comment]
Hi @xjl-0806  👋

Ensure you have a successful GitHub Education [verification status](https://education.github.com/discount_requests/pack_application) and have received the confirmation email that your Academic benefits have been granted - this usually arrives 72 hours after you have been approved.

To enable Copilot, go to your account settings and under Code, planning, and automation section, select Copilot. You'll see "You're eligible to use copilot for free" and a button to claim the offer.

If you still unable to enable Copilot please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "I am a verified student having trouble redeeming my Global Campus offers". 👍

Hope this helps! Let us know how you get along 👍
